src/ffmpeg.mk

```diff
@@ -37,8 +37,7 @@ define $(PKG)_BUILD
             --disable-static --enable-shared ) \
         --yasmexe='$(TARGET)-yasm' \
         --disable-debug \
-        --disable-pthreads \
-        --enable-w32threads \
+        --disable-$(if $(findstring posix, $(MXE_GCC_THREADS)),w32threads,pthreads) \
```

Comment:
Replace `MXE_GCC_THREADS ` test with either `WIN32_THREADS` or `POSIX_THREADS` tests.

Reply:
Ok

Reply:
@tonytheodore What about cases where I just want to check threading against no threading? Should I use `$(or)` or will `MXE_GCC_THREADS` be good for those?

Reply:
> or will MXE_GCC_THREADS be good for those

`MXE_GCC_THREADS` is compiler-specific, consider the case where it isn't set.

Short-circuit `$(or)` is ideal, though nested `ifs` may be more readable in the future.

Reply:
If you're unsure, default to pthreads. 

I'm pretty sure most w32threads are placeholders.